packages/codegen-sdk/src/print-connection.ts

```diff
@@ -51,7 +51,28 @@ function printAbstractConnection(): string {
 function printConnectionVariables(): string {
   return printLines([
     printComment(["Variables required for pagination", "Follows the Relay spec"]),
-    `export type ${Sdk.CONNECTION_TYPE}${Sdk.VARIABLE_TYPE} = { after?: string; before?: string }`,
+    `export type ${Sdk.CONNECTION_TYPE}${Sdk.VARIABLE_TYPE} = {
+      ${Sdk.CONNECTION_AFTER}?: string | null;
+      ${Sdk.CONNECTION_BEFORE}?: string | null;
+      ${Sdk.CONNECTION_FIRST}?: number | null;
+      ${Sdk.CONNECTION_LAST}?: number | null;
+    }`,
+  ]);
+}
+
+/**
+ * Function to default any connection variables that are rerquired by the api
+ */
+function printConnectionDefault(): string {
+  return printLines([
+    printComment(["Default connection variables required for pagination", "Defaults to 50 as per the Linear API"]),
```

Comment:
Add a function to the sdk to default the first and last connection variables to 50 if not set